setup.py

```diff
@@ -1,38 +1,49 @@
 #!/usr/bin/env python
 from setuptools.command.build_py import build_py as _build
+from setuptools.command.develop import develop as _develop
```

Comment:
Are these changes to setup.py supposed to be part of this PR?  They look unconnected.

Reply:
Yes.  I made them while debugging. Happy to open a new pr if it really matters to you.

Reply:
No, that's fine, just wanted to check.  I don't know much about setuptools.